backend/danswer/document_index/interfaces.py

```diff
@@ -16,6 +16,13 @@ class DocumentInsertionRecord:
     already_existed: bool
 
 
```

Comment:
The point of this new data class is to standardize chunk requests.
This may be combined with the other data type used for fetching chunks above and below in the future...